### PR TITLE
[TASK] Prepare the 8.5.0 release

### PR DESCRIPTION
**Pull request by oliverklee**
I have compiled the changes that seemed relevant from the git history.

### CODE REVIEW

**Review by JakeQZ — COMMENTED** [review]
I'm struggling to understand this.  I can't see any changes made since 8.4.0 apart from dependabot.  Was 8.4.0 release made from a separate branch?  If so, which branch?

**Comment by oliverklee** [comment]
I think I see now where the confusion comes from. :sli


I created the 8.4.0 release (or better, the [8.4.0 release documentation on GitHub](https://github.com/sabberworm/PHP-CSS-Parser/releases/tag/8.4.0)) yesterday because it was missing. Version 8.4.0 had actually been released (i.e., git-tagged, which is what is relevant for Packagist/Composer) [on December 11th](https://github.com/sabberworm/PHP-CSS-Parser/tags).

You can list the commits since the 8.4.0 tag with this command:

```bash
git log 8.4.0..HEAD
```

I've done my best to make sense of the commits and compile them into the changelog. (For the future, I'd like us to directly add the changelog entries together with their corresponding PRs.)

**Comment by oliverklee** [comment]
Oh, and the 8.4.0 released was tagged from this repo's default branch, i.e., `master`, I think.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,31 @@
...
+
+### Removed
+
+### Fixed
+
+## 8.4.1
+
+### Added
+
+- Add more unit tests
```
Comment (JakeQZ) [inline CHANGELOG.md:22]: The unit test additions had PR numbers #381 and #382.

**Review by JakeQZ — CHANGES_REQUESTED** [review]
> Version 8.4.0 had actually been released (i.e., git-tagged, which is what is relevant for Packagist/Composer) [on December 11th](https://github.com/sabberworm/PHP-CSS-Parser/tags).

Ah.  I see now.

> Oh, and the 8.4.0 released was tagged from this repo's default branch, i.e., `master`, I think.

I've gone through the commits since 8.4.0 and believe I've found 3 more fixes and one additional feature.  I've also found the PR numbers for the items you added to the changelog without one.  The other items you added seem correct.

Given there has actually been one new feature since 8.4.0 (#384), should the next release in fact be 8.5.0?

**Comment by oliverklee** [comment]
Done and repushed.

**Review by JakeQZ — APPROVED** [review]
All looks good now.

Sorry for delay in re-review - I didn't get any email notification (though did about the original PR, and also received a couple of other GitHub notification emails on the same day).

**Comment by JakeQZ** [comment]
Auto-deletion of merged branches appears not to be enabled.

**Comment by JakeQZ** [comment]
Also, I didn't see the 'Squash-and-Merge' option, 'Merge Branch' was the default, but maybe I should have looked for a dropdown on the button to change it - can't remember now if there was one.

**Comment by sabberworm** [comment]
@JakeQZ @oliverklee thank you both so much!

**Comment by oliverklee** [comment]
I've just tagged and released the new version: https://github.com/sabberworm/PHP-CSS-Parser/releases/tag/8.5.0

**Pull request merged**